### DISCUSSION TITLE
Hypixel api limit extension

### DISCUSSION THREAD

**Discussion by ethnss** [HypixelDev/PublicAPI#517] [Q&A]
Is there a way I can get a higher limit of requests per minute for the Hypixel api? Currently 120 queries/ 1 minute.

**Comment by mdashlw** [comment]
> API keys have a default rate limit of 120 requests per minute. If you require a higher limit please contact us via our support desk with an explanation for the increased limit and your Minecraft Account UUID.

You could try contacting support, however it's **extremely** unlikely you'd get it, so don't get your hopes up.
